app/src/main/java/com/kickstarter/libs/utils/KoalaUtils.java

```diff
@@ -101,33 +106,27 @@ private KoalaUtils() {}
     return MapUtils.prefixKeys(properties, prefix);
   }
 
-  public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project) {
-    return projectProperties(project, "project_");
-  }
-
-  public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project, final @NonNull String prefix) {
-    return projectProperties(project, null, prefix);
+  public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project, final @Nullable User loggedInUser) {
+    return projectProperties(project, loggedInUser, "project_");
   }
 
   public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project, final @Nullable User loggedInUser, final @NonNull String prefix) {
     final Map<String, Object> properties = new HashMap<String, Object>() {
       {
         put("backers_count", project.backersCount());
+        put("comments_count", project.commentsCount());
         put("country", project.country());
+        put("duration", Math.round(ProjectUtils.timeInSecondsOfDuration(project)));
         put("currency", project.currency());
         put("goal", project.goal());
-        put("pid", project.id());
+        put("has_video", project.video() != null);
```

Comment:
So if the `project.video = null` it won't put the value in the map?

Reply:
No, I'm putting in a boolean called "has_video", so it's either true or false.